### PR TITLE
[WIP] blog post about revision timeouts

### PR DESCRIPTION
**Pull request by nader-ziada**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

- add a blog post about revision timeouts 
-
-

/hold 
for changes in serving to be done: https://github.com/knative/serving/pull/12970

### CODE REVIEW

**Comment by abrennan89** [comment]
/hold

This seems like something that should be in the core docs instead of a blog post

File: blog/docs/articles/about-serving-revision-timeout.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "about serving revision timeout"
+linkTitle: "about serving revision timeout"
+Author: "[Nader Ziada](https://twitter.com/nziada)"
+Author handle: https://github.com/nader-ziada
+date: 2022-07-20
+description: "Serving revision timeout different fields"
+type: "blog"
+---
+
```
Comment (abrennan89) [inline blog/docs/articles/about-serving-revision-timeout.md:10]: we don't use this frontmatter anymore - please see the other files in `articles` for reference

Comment (nader-ziada) [reply blog/docs/articles/about-serving-revision-timeout.md]: fixed, thanks

**Comment by abrennan89** [comment]
@nader-ziada I'm currently working on some updates to revision docs as part of https://github.com/knative/docs/pull/5156 and if it's OK with you would like to include this content at a later time in the main revision docs, rather than having a separate blog about it. I think it makes more sense to keep core revision docs together and reserve blogs for articles, etc that are based on non-core integrations, longer examples etc rather than just discussion of config options and things

**Comment by nader-ziada** [comment]
we can some of the content in the Revision docs, but Dave and Evan suggested a blog post when this change is ready to announce this change in behaviour, [conversation](https://knative.slack.com/archives/CA4DNJ9A4/p1657829912646539)  in slack

**Comment by nader-ziada** [comment]
will move to core docs as per @abrennan89 recommendation 

/close

**Pull request closed without merging**